### ISSUE TITLE
[doc] dynamic data usage

### ISSUE THREAD

**Issue by ahuet67** [cerberustesting/cerberus-source#1916]
Hello,

I'm trying to use **_dynamic data in data lib to run my tests several times with different data_**.
I created a datalib and, as the doc said, I created a csv with each row is a value and a column a set of data. Example :
![image](https://user-images.githubusercontent.com/21305034/50274132-bdc71300-043c-11e9-8ca7-42542e8115e9.png)

Now I've tried to import it to cerberus in Datalib, I've choosen "CSV - Dynamic test data from CSV file".
After saving it, the subdata are : 
![image](https://user-images.githubusercontent.com/21305034/50274308-36c66a80-043d-11e9-92db-85cce54dea52.png)

But now I don't know how to iterate on my tests with each data of this datalib. I know it is possible but I don't know how. Can you help me?

Thank you all for your great job!

**Comment by vertigo17** [comment]
First of all you should revert columns and rows.
Your file should look like :
![screenshot_20181227_185233](https://user-images.githubusercontent.com/5342048/50497239-1c405280-0a36-11e9-9edc-64355ea12af6.png)
Still you can leave the header line as long as you don't forget to flag it when uploading.

Once uploaded, you need to create a property that points to the datalib created and specify the Length number to 2 you you want to load the 2 rows.
Then loop on a step twice and use the variable %PROPERTY.n.SUBDATA1% 
replacing n by %system.CURRENTSTEP_INDEX%

**Comment by ahuet67** [comment]
Amazing!! Thank you very much for your feedback Vertigo.
I'm going to explain what I did in details.

I've called my datalib test-csv in the same system as my test.
In my property, I've just specified my datalib name and the length to number of rows (3).

in my data library (with a header and with "Automatically feed subdata's list with the first line of csv" checked):
![image](https://user-images.githubusercontent.com/21305034/50513309-4ece6780-0a97-11e9-80ef-6a4227f10932.png)

in my property :
![image](https://user-images.githubusercontent.com/21305034/50513333-758c9e00-0a97-11e9-9ebd-757418b40ad2.png)

In your step, add Step Loop with "whileConditionTrueDo", "IfNumericMinorOrEqual", "%system.CURRENTSTEP_INDEX%", "3".

in the step where I want to access the value, I've pasted 
``` %property.csvFromDatalib.%system.CURRENTSTEP_INDEX%.SUBDATA1% ```

Thank you again for you're time and great job!
